_posts/2020-01-23-operating-system-and-browser-accessibility-display-modes.md

```diff
@@ -0,0 +1,373 @@
+@media (prefers-color-scheme: dark) {
+  body {
+    background-color: #000000;
+    color: #ffffff;
+  }
+}
+```
+
+## Increased Contrast Mode
+
+This mode is only available on macOS and iOS. It will increase the difference in color values between the gray colors used for the OS visuals, as well as drawing more obvious borders between different UI elements.
+
+This is helpful for:
+
+- People experiencing low vision conditions,
+- and/or who have lower quality displays,
+- and/or less technologically literate people, who may need the difference between different parts of the OS visuals to be more obvious.
+
+An example of one of these conditions could be vision-related issues caused by Glaucoma. Here, the increased contrast and borders may help a person who has blurry central vision and a loss of peripheral vision understand the boundaries of different parts of the interface.
+
+### What it affects
+
+Increased Contrast Mode will update **all** User Interface (<abbr>UI</abbr>) elements of macOS and iOS. Most apps developed by Apple will also support this mode. Third party apps that use Apple’s interface library will also support this mode, but custom, nonstandard UI elements will **not** be supported. Websites and web apps do **not** support this mode.
+
+<figure role="figure" aria-label="Preview.app comparing regular and Increased Contrast Mode.">
+  <img alt="A comparison of regular and Increased Contrast Mode UI. The bottom screenshot is Preview in regular mode. The buttons on the app’s toolbar are light gray gradient on a medium gray background. The top screenshot is Preview in Increased Contrast Mode. The buttons on the app’s toolbar are solid white with a black outline. A black outline has also been applied to the overall app window border. Preview is displaying a photo of a Mallard duck." src="/img/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/increased-contrast-mode.png">
+  <figcaption>Preview.app comparing regular and Increased Contrast Mode.</figcaption>
+</figure>
+
+### How to enable it
+
+Assumes the latest version of each Operating System or browser.
+
+#### iOS
+
+1. Go to **Settings**.
+1. Go to **Accessibility**.
+1. Go to **Display & Text Size**.
+1. Set the **Increase Contrast** toggle to the **on** position.
+
+#### macOS
+
+1. Go to **System Preferences**.
+1. Go to **Accessibility**.
+1. Go to **Display**.
+1. Check **Increase contrast**.
+
+## Inverted Colors Mode
+
+When Inverted Colors mode is activated, the color of every pixel on the screen is reversed. Black becomes white, red becomes turquoise, green becomes purple, etc.
+
+Because it affects all content displayed on the screen, Inverted Colors mode can be very helpful in situations where a light-colored interface does not provide Dark Mode support. Here, Inverted Colors mode can force a bright UI into kind of jury-rigged Dark Mode. Keep in mind that the opposite also applies. A dark UI will turn light if Inverted Colors Mode is active.
```

Comment:
you capitalize 'mode' in other sections and when you introduced "Inverted Colors Mode" so that should be done here too.

Reply:
Ah, good catch. Thank you!